### PR TITLE
fix(hdfs): Include the connection failure reason

### PR DESCRIPTION
**Pull request by Yohahaha**

### CODE REVIEW

**Comment by Yohahaha** [comment]
@JkSelf please help take a look, thank you!

**Comment by JkSelf** [comment]
> @JkSelf please help take a look, thank you!

@Yohahaha Thanks for your fix. Can you help to figure out the different error message in the PR description? Thanks.

File: velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.cpp [hunk 1]
```diff
@@ -31,7 +31,7 @@ class HdfsFileSystem::Impl {
       const HdfsServiceEndpoint& endpoint) {
     auto status = filesystems::arrow::io::internal::ConnectLibHdfs(&driver_);
     if (!status.ok()) {
-      LOG(ERROR) << "ConnectLibHdfs failed ";
+      LOG(ERROR) << "ConnectLibHdfs failed " << status.ToString();
```
Comment (majetideepak) [inline velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.cpp:34]: @Yohahaha nit: can you add a colon? `"ConnectLibHdfs failed: "`

**Comment by Yohahaha** [comment]
@majetideepak would you help review again? thank you!

**Pull request closed without merging**